create-milestone/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Create Milestone Regularly'
+description: 'Manupate Github API'
+inputs:
```

Comment:
The inputs should really just match `createMilestone`'s arguments with some variables to template the value of each